content/en/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information.md

```diff
@@ -1,67 +1,69 @@
 ---
-title: Expose Pod Information to Containers Through Files
```

Comment:
Please fix the frontmatter back up and revert the shortcodes to use the Hugo syntax throughout this file too.